pkg/spoke/submarineragent/config_controller.go

```diff
@@ -186,13 +191,29 @@ func (c *submarinerConfigController) sync(ctx context.Context, syncCtx factory.S
 		return updateErr
 	}
 
+	return c.syncConfig(ctx, recorder, config)
```

Comment:
The commit message says that its for GCP Clusters, but the code seems to be applicable for the other clusters as well. Can you please clarify/update the commit message?

Reply:
Sure I updated the commit message